scripts/monitoring/cron-send-security-updates-count.py

```diff
@@ -82,7 +82,7 @@ def get_security_updates(self):
             self.sec_updates = []
             raise subprocess.CalledProcessError(proc.returncode, subprocess.list2cmdline(cmd), output=stdout+stderr)
         lineslist = [line for line in stdout.split('\n') if line and 'Errno' not in line]
-        self.sec_updates = [UpdateItem(*re.split(r'\s+', line)) for line in lineslist]
+        self.sec_updates = [UpdateItem(*re.split(r'\s+', line)) for line in lineslist if re.split(r'\s+', line) == 3]
```

Comment:
What output did you see that didn't split into 3 fields?

I recall seeing "Could not connect" type errors.  The concern with ignoring these is that first - if an actual error message DOES split into three fields it could be incorrectly processed.  And second - while I don't fundamentally object to throwing away the details of lines that don't split into three fields, we can't throw away the fact that an unknown error was found in the output.  As such, our list of errata cannot be trusted to be correct and should be flagged as such.

tl;dr - no fundamental objections, but handle the case of "threw away data" as an error in itself to report.